civitai-downloader-v2/src/monitoring/metrics_collector.py

```
@@ -0,0 +1,375 @@
+        
+        with self._gauges_lock:
+            total_gauges = len(self._gauges)
+            recent_gauges = [
+                (k, v) for k, v in self._gauges.items()
+                if self._last_update.get(k, 0) > time.time() - 300  # Last 5 minutes
+            ]
+        
+        with self._histograms_lock:
+            total_histograms = len(self._histograms)
+            active_histograms = [
+                k for k in self._histograms.keys()
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)